src/pyramid_elasticapm/__init__.py

```diff
@@ -26,23 +26,21 @@ def elasticapm_instrument(event):
     elasticapm.instrument()
 
 
+# https://www.elastic.co/guide/en/apm/agent/python/current/configuration.html
+ELASTICAPM_PREFIX = 'elasticapm.'
+
+
 class TweenFactory:
     def __init__(self, handler, registry):
         self.handler = handler
         self.registry = registry
         settings = registry.settings
 
         config = {
-            'SERVICE_NAME': settings['elasticapm.service_name'],
-            'SERVER_URL': settings['elasticapm.server_url'],
-            'SECRET_TOKEN': settings['elasticapm.secret_token'],
-            'ENVIRONMENT': settings['elasticapm.environment'],
-            'TRANSACTION_SAMPLE_RATE': settings['elasticapm.transaction_sample_rate'],
+            key.replace(ELASTICAPM_PREFIX, '').upper(): value
+            for key, value in settings.items()
+            if key.startswith(ELASTICAPM_PREFIX)
         }
-        if settings.get('elasticapm.transactions_ignore_patterns', ''):
-            config['TRANSACTIONS_IGNORE_PATTERNS'] = settings[
-                'elasticapm.transactions_ignore_patterns'
-            ].split()
```

Comment:
I don't understand this...

This looks like the setting needs to be split into a list. I can't see how this behaviour is preserved....
Could you explain it to me? _maybe it's a Monday morning thing... :)_

Reply:
I nearly had the implementation ready, just to find that the elastic client already handles that case of an unsplit value:
![image](https://github.com/bettermarks/pyramid-elasticapm/assets/135657/555fc2ca-90c1-4cb6-ac4b-742648377812)
so I will not add any additional logic to this PR